src/libs/svg/lv_svg_render.h

```diff
@@ -11,6 +11,10 @@
  *********************/
 #include "../../lv_conf_internal.h"
 
+#if !LV_USE_VECTOR_GRAPHIC
```

Comment:
`#if LV_USE_SVG` was deleted, but it still should be there.

Reply:
omg. sorry.

Reply:
and I see the entire `#if` below it can now go